boutdata/mms.py

```diff
@@ -612,7 +613,7 @@ def write(self, nx, ny, filename, MXG=2):
 
         hthe = zeros([ngx, ngy])
 
-        I = zeros([ngx, ngy])
+        # I = zeros([ngx, ngy])
```

Comment:
Could be removed if not used?

Reply:
If anyone wants to complete implementation of the `write()` method, then it probably would be needed, so I commented it out just to skip the `flake8` errors. Arguably we should remove non-implemented functions...

Reply:
:+1: for removing. It can be restored from the history, if it is wanted ...